OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -318,5 +320,15 @@ public FrozenActor FromID(uint id)
 
 			return fa;
 		}
+
+		public IEnumerable<FrozenActor> FrozenActorsInBox(WPos a, WPos b)
+		{
+			// Find bounding cell coordinates
+			var left = Math.Min(a.X, b.X) / 1024;
+			var top = Math.Min(a.Y, b.Y) / 1024;
+			var right = (Math.Max(a.X, b.X) + 1023) / 1024;
+			var bottom = (Math.Max(a.Y, b.Y) + 1023) / 1024;
```

Comment:
Why add an almost full cell here?
I assume this expects that the division and rounding will take care of the excess, while the extra value is to fill the cell since the caller uses `CenterOfCell`? But we really should be able to handle that better.